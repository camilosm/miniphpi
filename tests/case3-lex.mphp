/* isso
eh um comentario incompleto