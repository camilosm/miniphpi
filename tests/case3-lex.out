03: Fim de arquivo inesperado
